HandyHansel/Program.cs

```diff
@@ -58,14 +58,14 @@ public static void ConfigureServices(HostBuilderContext ctx, IServiceCollection
                 Pooling = hangfireConfig.Database.Pooling,
             };
 
-            services
+            _ = services
                 .AddSingleton<IBotAccessProviderBuilder, BotAccessProviderBuilder>()
                 .AddSingleton<BotService>()
                 .AddHangfire(configuration => configuration
                     .SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
                     .UseSimpleAssemblyNameTypeSerializer()
                     .UseRecommendedSerializerSettings()
-                    .UsePostgreSqlStorage(connectionStringBuilder.ConnectionString))
+                    .UsePostgreSqlStorage(connectionStringBuilder.ConnectionString, new PostgreSqlStorageOptions { DistributedLockTimeout = TimeSpan.FromMinutes(1), }))
```

Comment:
Was getting weird exception being thrown. This is needed to fix it. I need to better understand why.